spec.html

```diff
@@ -4078,36 +4078,36 @@ <h1>Type Conversion</h1>
     <p>The BigInt type has no implicit conversions in the ECMAScript language; programmers must call BigInt explicitly to convert values from other types.</p>
 
     <emu-clause id="sec-toprimitive" aoid="ToPrimitive" oldids="table-9">
-      <h1>ToPrimitive ( _input_ [ , _PreferredType_ ] )</h1>
-      <p>The abstract operation ToPrimitive takes argument _input_ and optional argument _PreferredType_. It converts its _input_ argument to a non-Object type. If an object is capable of converting to more than one primitive type, it may use the optional hint _PreferredType_ to favour that type. It performs the following steps when called:</p>
+      <h1>ToPrimitive ( _input_ [ , _preferredType_ ] )</h1>
+      <p>The abstract operation ToPrimitive takes argument _input_ and optional argument _preferredType_. It converts its _input_ argument to a non-Object type. If an object is capable of converting to more than one primitive type, it may use the optional hint _preferredType_ to favour that type. It performs the following steps when called:</p>
       <emu-alg>
         1. Assert: _input_ is an ECMAScript language value.
         1. If Type(_input_) is Object, then
-          1. If _PreferredType_ is not present, let _hint_ be *"default"*.
-          1. Else if _PreferredType_ is hint String, let _hint_ be *"string"*.
+          1. If _preferredType_ is not present, let _hint_ be *"default"*.
```

Comment:
Since `_hint_` is now only used in the `If _exoticToPrim_ is not *undefined*` branch, this if/else if/else should probably be moved under that.

Reply:
Done.